modules/ROOT/pages/managing-schedules.adoc

```diff
@@ -4,33 +4,39 @@ include::_attributes.adoc[]
 
-* You have a job that runs nightly but you want to trigger it right now so you can debug it. You want to do this in runtime so you don’t need to update the schedule in your underlying application.
+* You have a job that runs nightly but you want to debug it now.
++
+You can trigger it while it's running without updating the schedule in your underlying application.
 * You want to change the schedules in your application at runtime without redeploying.
 * You want to disable a scheduled job while one of the applications you are connecting to is undergoing maintenance.
 * You want runtime visibility into the schedules in your application.
 
 == Considerations
 
+When using the *Schedules* tab, consider the following:
+
 * Schedulers are managed by CloudHub and are triggered in only one worker even if more are available.
-* Setting a polling Schedule for less than a 10-second frequency between calls is not recommended.
+* Setting a polling schedule for less than a 10-second frequency between calls is not recommended.
 * If a scheduled job is not triggered one or more times because the application is down, CloudHub triggers the job as soon as the application restarts.
-* CloudHub scheduler reads a schedule job configuration every time it runs. This means that if you update the schedule time, the change is not immediate. The job executes as it was initially configured before the scheduler picks up the next setting for the next time it runs. To apply your changes immediately, run your jobs after every schedule update.
+* CloudHub scheduler reads a schedule job configuration every time it runs.
++
+If you update the schedule, the change is not immediate. The job executes as it was initially configured before the scheduler picks up the next setting for the next time it runs. To apply your changes immediately, run your jobs manually after updating the schedule.
 * If poll execution time takes longer than 12 hours, the scheduler fails and is re-executed.
-* CloudHub schedules cannot be managed using scripts. Use the CloudHub API or Schedules tab in Runtime Manager to manage CloudHub schedules.
-
+* CloudHub schedules cannot be managed using scripts.
++
+Use the CloudHub API or *Schedules* tab in Runtime Manager to manage CloudHub schedules.
 
-[NOTE] 
-Flow names can contain only letters (a-zA-Z), numbers (0-9), hyphen (-), underscore (_), and period (.).
+// Flow names can contain only letters (a-zA-Z), numbers (0-9), hyphen (-), underscore (_), and period (.).
 
 == View Application Schedules in the Console
 
-To use the scheduling feature in Runtime Manager, deploy an application that includes one or more polling elements. For example, the application illustrated below includes two simple flows, each with a polling scope.
+To use the scheduling feature in Runtime Manager, deploy an application that includes one or more polling elements. This example includes two simple flows, each with a polling scope.
```

Comment:
Consider ending with a colon to point directly to a graphic element, and/or changing "This" to "The following."